src/css/profile/mobile/common/timepicker.less

```diff
@@ -22,6 +22,28 @@
 		}
 	}
 
+	&[data-format="24"] {
+		.ui-time-picker-container {
+			width: 30%;
+
+			&-hour::after {
+				content: ":";
+				height: 164 * @px_base;
+				width: 40 / 3%;
```

Comment:
It looks incorrect... is this works?

Reply:
Yes,, it works,, Wouldn't it be strange to use 'calc'?
According to guide(4-15), Time Spin has 30% for the width. So, the remaining area(40%) is divided into three.

Reply:
I don't see the `calc` and I don't understand this calculation.

Reply:
Yes, it is just the 40 over 3. 40/3 = 13.333333 %.

Reply:
maybe you wanted to write:
`calc(40% / 3)`

Reply:
Right.
Done.